lib/board.rb

```diff
@@ -0,0 +1,52 @@
+class Board
+ attr_accessor :cells
+ @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+   def initialize
+ 	   reset!
+   end
+ 
+ def reset!
+  @cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "] 
+ end
+ 
+ def display
+ puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+ puts "-----------"
+ puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+ puts "-----------"
+ puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+ end
+ 
+ def position(input)
+ 	 cells[input.to_i - 1]
+  end
+
+  def update(input, player)
+    cells[input.to_i - 1] = player.token
+  end
+ 
+  def full?
+    if cells.all? {|value| value == "X" || value == "O"}
```

Comment:
Because the expression will evaluate to a boolean, you can use the implicit return value; so the entire method can be 
`cells.all? {|value| value == "X" || value == "O"}`